src/routes/Sidebar.svelte

```diff
@@ -74,38 +77,40 @@
         </Command.Group>
         <Separator />
         <Command.Group>
-          <AlertDialog.Root>
```

Comment:
Moved the dialog out from here, otherwise it got unmounted when the popover close which meant that you couldn't focus the input.